vs-extension.shared/CorDebug/CorDebugFunction.cs

```diff
@@ -8,25 +8,26 @@
 using nanoFramework.Tools.Debugger;
 using System;
 using System.Collections;
+using System.Collections.Generic;
 using System.Diagnostics;
 
 namespace nanoFramework.Tools.VisualStudio.Extension
 {
-    public class CorDebugFunction : ICorDebugFunction , ICorDebugFunction2
+    public class CorDebugFunction : ICorDebugFunction, ICorDebugFunction2
     {
-        CorDebugClass    m_class;
-        Pdbx.Method      m_pdbxMethod;
-        CorDebugCode     m_codeNative;
-        CorDebugCode     m_codeIL;
-        uint             m_tkSymbolless;
+        CorDebugClass m_class;
```

Comment:
I know at this point it's very tiny regarding all the work done: we should be explicit with the visibility. Feel free to ignote!